web/modules/custom/rdf_entity/src/RdfPermissions.php

```diff
@@ -41,19 +41,25 @@ public function rdfTypePermissions() {
    */
   protected function buildPermissions(RdfEntityType $type) {
     $type_id = $type->id();
-    $type_params = array('%type_name' => $type->label());
+    $type_params = ['%type_name' => $type->label()];
 
-    return array(
-      "create $type_id rdf entity" => array(
+    return [
+      "create $type_id rdf entity" => [
```

Comment:
And should probably be changed (There is already a commit to change this in the other workflows).
I know that it really is much better to be an "rdf entity" though if we change it to use the entity type id, it is going to help us in cases where we cant to auto generate some permissions like in the comment above.
So sacrifice some minor readability instead of generating duplicated classes only to override one parameter.
And I am aware that nodes are changing this to 'content' but for example og is following the pattern below.

```
    $operations = [
      [
        'name' => "create $group_content_bundle_id $group_content_entity_type_id",
        'title' => $this->t('Create %bundle @entity', $args),
        'operation' => 'create',
      ],
```
It will really help us generating automatically the strings.

P.S. there are discussions changing the entitiy id itself in the end.